rocky/rocky/views/organization_member_list.py

```diff
@@ -87,7 +87,7 @@ def handle_page_action(self, action: str):
             else:
                 raise Exception(f"Unhandled allowed action: {action}")
             organizationmember.save()
-        except RequestException as exception:
+        except RequestError as exception:
```

Comment:
I don't understand this one because the code doesn't do any request?

Reply:
I believe it has a signal attached to it that does requests